packages/localizations/src/en-US.ts

```diff
@@ -325,6 +325,11 @@ export const enUS: LocalizationResource = {
         subtitleNewTab: 'Return to the newly opened tab to continue',
         titleNewTab: 'Signed in on other tab',
       },
+      clientMismatch: {
+        subtitle:
+          'Open the verification link on the device and browser from which you initiatied the sign in to continue',
```

Comment:
```suggestion
          'To continue, open the verification link on the device and browser from which you initiated the sign-in',
```

Typo fix, use the proper noun for sign-in (sign in is the verb), and move the "to continue" to start.

Reads better to me :)

Reply:
Reads better to me as well, thanks!